site/index.html

```
@@ -2311,11 +2311,17 @@ <h3>Browser-session setup</h3>
       <div class="community-plugin-desc">Nightrider visualizer</div>
       <div class="community-plugin-author"><a href="https://github.com/HANCORE-linux">@HANCORE-linux</a></div>
     </div>
+    <div class="community-plugin">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div></div> ], start tag match failed [ <div class="community-plugin"> ] on line 2314.

(tag-pair)